specifyweb/frontend/js_src/lib/components/QueryBuilder/Line.tsx

```diff
@@ -230,6 +232,11 @@ export function QueryLine({
     availableFilters.length > 1 || availableFilters[0][0] !== 'any';
 
   const hasAny = field.filters.some(({ type }) => type === 'any');
+  const [collapse, setCollapse] = React.useState(true);
```

Comment:
rename `collapse` to `isCollapsed` or something like that - that's a more appropriate name for a boolean

`collapse` makes it seem like it's a function that will collapse when called rather than a boolean indicating whether we are collapsed right now

Thus, generally it's a good practice to make boolean variable names start with `is`